### PR TITLE
Update stress chaos experiment doc with stress-ng option details

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
Add stress-ng options used under the hood for cpu and memory stress.

### CODE REVIEW

**Review by cwen0 — APPROVED** [review]

File: docs/chaos_experiments/stress_chaos.md [hunk 1]
```diff
@@ -14,24 +14,24 @@ A StressChaos shares common configurations like other chaos, such as how to sele
...
   1. `memory`
 
      A `memory` stressor will continuously stress virtual memory out.
 
-     | Option    | Type    | Required | Description                              |
-     | --------- | ------- | -------- | ---------------------------------------- |
-     | `workers` | Integer | True     | Specifies concurrent stressing instance. |
+     | Option    | Type    | Required | Description                                                                           |
+     | --------- | ------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------|
+     | `workers` | Integer | True     | Start  N workers that grow their heaps by reallocating memory. For more details refer `--bigheap` option on `stress-ng` man page. |
```
Comment (cwen0) [inline docs/chaos_experiments/stress_chaos.md:25]: We have submitted a pr https://github.com/chaos-mesh/chaos-mesh/pull/1581 to revert `Use bigheap to inject heap stress`.  This description should be updated.

Comment (anuragpaliwal80) [reply docs/chaos_experiments/stress_chaos.md]: sure.

Comment (anuragpaliwal80) [reply docs/chaos_experiments/stress_chaos.md]: @cwen0 can you please review again as the suggested changes have been done.

**Comment by anuragpaliwal80** [comment]
@cwen0 Can you please review ?

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**